config/config.go

```diff
@@ -377,7 +377,7 @@ func (t *TCF2) PurposeEnforcingVendors(purpose consentconstants.Purpose) (value
 
 // PurposeVendorExceptions returns the vendor exception map for a given purpose if it exists, otherwise it returns
 // an empty map of vendor exceptions
-func (t *TCF2) PurposeVendorExceptions(purpose consentconstants.Purpose) (value map[openrtb_ext.BidderName]struct{}) {
+func (t *TCF2) PurposeVendorExceptions(purpose consentconstants.Purpose) (vendorExceptionMap map[openrtb_ext.BidderName]struct{}) {
```

Comment:
Nitpick: maybe just call this `vendorExceptions` so the type isn't embedded in the name?